src/LumenServiceProvider.php

```diff
@@ -27,7 +27,7 @@ public function register()
     public function boot()
     {
         if ($this->app['config']->get('snappy.pdf.enabled')) {
-            $this->app['snappy.pdf'] = $this->app->share(function ($app) {
+            $this->app['snappy.pdf'] = $this->app->singleton(function ($app) {
```

Comment:
This is incorrect, should be just `$this->app->singleton('snappy.pdf', function ($app) {`)